docs/source/design/testing.rst

```diff
@@ -0,0 +1,99 @@
+.. -*- mode: rst -*-
+.. vi: set ft=rst sts=4 ts=4 sw=4 et tw=79:
+
+.. _chap_design_testing:
+
+**********************************
+Continuous integration and testing
+**********************************
+
+.. topic:: Specification scope and status
+
+   This specification describes the current implementation.
+
+DataLad is tested using a pytest-based testsuite that is run locally and via continuous integrations setups.
+Code development should ensure that old and new functionality is appropriately tested.
+The project aims for good unittest coverage (at least 80%).
+
+Running tests
+=============
+
+``datalad/tests`` contains tests for the core portion of the project.
+More tests are provided under corresponding submodules in ``tests/`` subdirectories to simplify re-running the tests concerning that portion of the codebase.
```

Comment:
```suggestion

```

Reply:
Suggestion belongs to the one above.

Change suggested, because of phrasing. To single out the toplevel tests as "core portion" is misleading. Tthey test some general mechanisms of the entire framework but basically no actual datalad functionality and on top it's confusing because we do have an actual module named "core".